tools/make/data.toml

```diff
@@ -78,6 +78,20 @@ args = [
     "--bin=make-testdata-legacy",
 ]
 
+[tasks.testdata-legacy]
+description = "Verify the data for the legacy icu_testdata crate"
+category = "ICU4X Data"
+dependencies = ["testdata-legacy-gen"]
+script_runner = "@duckscript"
+script = """
+    exec --fail-on-error cargo clean # free up space
```

Comment:
not sure we want to automatically `cargo clean`

Reply:
reusing it instead